src/components/AppLink/AppLink.test.tsx

```diff
@@ -0,0 +1,78 @@
+import { screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import type { RootState } from "store/store";
+import { rootStateFactory, jujuStateFactory } from "testing/factories";
+import { modelListInfoFactory } from "testing/factories/juju/juju";
+import { renderComponent } from "testing/utils";
+
+import AppLink from "./AppLink";
+
+describe("AppLink", () => {
+  let state: RootState;
+
+  beforeEach(() => {
+    state = rootStateFactory.withGeneralConfig().build({
+      juju: jujuStateFactory.build({
+        models: {
+          abc123: modelListInfoFactory.build({
+            uuid: "abc123",
+            name: "test-model",
+            ownerTag: "user-eggman@external",
+          }),
+        },
+      }),
+    });
+  });
+
+  it("should render correctly when there is no view", () => {
+    renderComponent(
+      <AppLink uuid="abc123" appName="mockApp">
+        abc123
+      </AppLink>,
+      { state }
+    );
+    expect(screen.queryByRole("link", { name: "abc123" })).toHaveAttribute(
+      "href",
+      "/models/eggman@external/test-model/app/mockApp"
```

Comment:
I think it would be nice if all these tests used the URL definitions, so that if the URLs ever change the tests will continue to work.

```suggestion
      urls.model.app.index({ userName: "eggman@external", modelName: "test-model", appName: "mockApp" })
```

Reply:
Thanks for the review! Changed this in my latest commit.